invenio_records_lom/fixtures/demo.py

```diff
@@ -360,6 +364,40 @@ def create_fake_record(fake: Faker) -> dict:
     return json.loads(json.dumps(data_to_use))
 
 
+def create_fake_record(fake: Faker):
```

Comment:
add this method to fixtures/__init__.py, so it can be used in cli.py without importing the demo module directly.